src/dsmr/parser.h

```diff
@@ -83,7 +87,11 @@ struct ParsedData<> {
 // Do not use F() for multiply-used strings (including strings used from
 // multiple template instantiations), that would result in multiple
 // instances of the string in the binary
+#ifndef ARDUINO_ARCH_ESP8266
 static constexpr char DUPLICATE_FIELD[] PROGMEM = "Duplicate field";
+#else
+static constexpr char DUPLICATE_FIELD[] = "Duplicate field";
```

Comment:
> Apparently just using F() doesn't disable string pooling for AVR and ARM (at least for Arduino Due) as described by:

Hm, that's not my experience. On AVR, using F() prevents string pooling (at least with current compiler versions, see https://github.com/arduino/Arduino/issues/4863). On the Due (SAM core) this does not apply, since I think PROGMEM is essentially a no-op there (since flash is mapped into the RAM space). I'm not sure about how ESP works in this regard.

You referenced an issue in the ESP core, but that's about the `DISABLED` macro, not the `PROGMEM` issue. Did you perhaps link the wrong one?